LibSWBF2.NET/Wrappers/Segment.cs

```diff
@@ -0,0 +1,162 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Runtime.InteropServices;
+
+using LibSWBF2.Logging;
+using LibSWBF2.Enums;
+using LibSWBF2.Utils;
+
+namespace LibSWBF2.Wrappers
+{
+    [StructLayout(LayoutKind.Sequential, Pack=1)]
+    public struct VertexWeight 
+    {
+        public float weight;
+        public byte index;
+    }
+
```

Comment:
Like the ```Vector``` cpp structs, I added alignment/packing to the ```VertexWeight``` cpp struct for simple marshaling.